indico/modules/events/editing/client/js/management/editable_type/EditableList.jsx

```diff
@@ -438,30 +453,85 @@ function EditableListDisplay({
       />
     );
   };
+  const renderTags = (__, editable) => {
+    const tags = editable.tags.sort((a, b) => a.code.localeCompare(b.code));
+    const labelTags = () => {
+      return (
+        <div style={{display: 'flex', flexWrap: 'nowrap'}}>
+          {tags.map(t => {
+            return (
+              <Label key={t.code} size="small" color={t.color}>
+                {t.code}
+              </Label>
+            );
+          })}
+        </div>
+      );
+    };
+
+    const colorTags = () => {
+      return (
+        <div>
+          {tags.map(t => {
+            return <Label key={t.code} circular empty color={t.color} />;
+          })}
+        </div>
+      );
+    };
+
+    return [labelTags(), colorTags()];
+  };
   const renderFuncs = {
     friendlyId: renderId,
     title: renderTitle,
     code: renderCode,
     revision: editable => (editable ? editable.revisionCount : ''),
     status: renderStatus,
     editor: renderEditor,
+    tags: renderTags,
   };
 
   // eslint-disable-next-line react/prop-types
   const renderCell = ({dataKey, rowIndex}) => {
     const row = sortedList[rowIndex];
     const fn = renderFuncs[dataKey];
-    return (
-      <TooltipIfTruncated>
-        <div
-          styleName="rowcolumn-tooltip"
-          role="gridcell"
-          style={dataKey === 'title' ? {display: 'block'} : {}}
-        >
-          {fn(row[dataKey], row.editable, rowIndex)}
-        </div>
-      </TooltipIfTruncated>
-    );
+
+    if (dataKey !== 'tags') {
+      return (
+        <TooltipIfTruncated>
+          <div
+            styleName="rowcolumn-tooltip"
+            role="gridcell"
+            style={dataKey === 'title' ? {display: 'block'} : {}}
+          >
+            {fn(row[dataKey], row.editable, rowIndex)}
+          </div>
+        </TooltipIfTruncated>
+      );
+    } else if (row.editable?.tags) {
+      const [labels, colors] = fn(row[dataKey], row.editable, rowIndex);
```

Comment:
The single-tag popup is a bit useless right now:

![image](https://github.com/user-attachments/assets/d1032958-04e3-4d9b-b572-60ff26e5fb9a)


How about always showing the long version of the tag ("code: text") in the popup?